src/main/java/net/bramp/ffmpeg/builder/FFmpegOutputBuilder.java

```diff
@@ -223,6 +225,23 @@ public FFmpegOutputBuilder setComplexVideoFilter(String filter) {
     return this;
   }
 
+  /**
+   * Allows to add metadata to output video. Which keys are possible depends on the used output
+   * video codec.
+   * 
+   * @param key Metadata key, e.g. "comment"
+   * @param value Value to set for key
+   * @return This instance
+   */
+  public FFmpegOutputBuilder addMetaTag(String key, String value) {
+    this.video_enabled = true;
+    if (this.meta_tags == null) {
+      this.meta_tags = new ArrayList<>();
+    }
+    this.meta_tags.add(key + "=" + value);
```

Comment:
It is best to wrap the whole "key=value" thing in quotes, so the output would be 

    -metadata "key=value"

If that is not done automatically, I will gladly wrap it in quotes when inserting.